src/classes/trip.js

```diff
@@ -1,11 +1,11 @@
 import time from '../scripts/time';
 
 class Trip {
-  constructor(tripInfo, destination) {
-    this.tripLocation = tripInfo.destinyID;
+  constructor(tripInfo, price, destination) {
     this.tripStatus = time.isBetween(tripInfo.status, tripInfo.date.split('/'), time.daysFromDate(tripInfo.date.split('/'), tripInfo.duration));
     this.year = tripInfo.date.split('/')[0];
-    this.destination = destination;
+    this.price = price;
+    this.destination = {location: destination.destination, image: destination.image, alt: destination.alt}
```

Comment:
Decided to house the destination on the Trip itself, since it would be a little redundant to have a class for that when we are never modifying that data